### PR TITLE
Rh ajax intro

### PR DESCRIPTION
**Pull request by rezahusain**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -46,9 +48,11 @@ def update
       if @comment.update(comment_params)
         format.html { redirect_to root_url, notice: "Comment was successfully updated." }
         format.json { render :show, status: :ok, location: @comment }
+        format.js
       else
         format.html { render :edit, status: :unprocessable_entity }
         format.json { render json: @comment.errors, status: :unprocessable_entity }
+
```
Comment (ahuynh3a) [inline app/controllers/comments_controller.rb:55]: nitpik: please be mindful of unnecessary white spaces

File: app/views/comments/_form.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<li class="list-group-item">
-  <%= form_with(model: comment) do |form| %>
+<li id="<%= dom_id(comment.photo) %>_<%= dom_id(comment) %>_form" class="list-group-item">
+  <%= form_with(model: comment, local: false) do |form| %>
```
Comment (ahuynh3a) [inline app/views/comments/_form.html.erb:2]: Great job of being mindful of when to use local:false and remote:true

File: app/views/follow_requests/create.js.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+var current_state = $(".<%=dom_id(@follow_request.sender)%>_<%=dom_id(@follow_request.recipient)%>");
```
Comment (ahuynh3a) [inline app/views/follow_requests/create.js.erb:1]: great job using variables to make your code more legible.

**Review by ahuynh3a — APPROVED** [review]
Great job Reza! Your code looks great! It is easy to read and works well!

**Review by APatel-AI — APPROVED** [review]
Nice Job Reza!!, The code is well structured and concise.

**Pull request merged**